src/main.rs

```diff
@@ -37,6 +37,10 @@ struct Cli {
     command: Option<Commands>,
     #[clap(short, long)]
     brokers: String,
+    #[clap(short = 'a', long, default_value = "0.0.0.0")]
```

Comment:
That's generally not a good default but doesn't matter for client-swarm

Reply:
I'll change it to the local loopback for the default and change it to all addresses in ducktape.